src/attr/_make.py

```diff
@@ -1582,6 +1658,9 @@ def _add_repr(cls, ns=None, attrs=None):
 def _make_init(
     cls, attrs, post_init, frozen, slots, cache_hash, base_attr_map, is_exc
 ):
+    """
+    Add an init method to *cls*.
```

Comment:
That's not true, is it? I only generates the method and returns it?

Reply:
Fixed.